src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Lists/BitBasicListTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Bunit;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using static Bit.Client.Web.BlazorUI.Tests.Lists.BitBasicListTest;
+
+namespace Bit.Client.Web.BlazorUI.Tests.Lists
+{
+    [TestClass]
+    public class BitBasicListTests : BunitTestContext
+    {
+
+        [DataTestMethod, DataRow(1000)]
+        public async Task BitBasicListShoudReturnExpectedChildElements(int itemCount)
+        {
+            var component = RenderComponent<BitBasicListTest>();
+
+            var bitList = component.Find(".bit-basic-list");
+            int comItemCount = bitList.ChildElementCount;
+
+            Assert.AreEqual(comItemCount, itemCount + 1);
```

Comment:
why itemCount+1 ?

Reply:
there was a mistake and should be +2,
When we use virtualization, two additional parts are created, that's why this method use itemCount +2